app/assets/v2/js/status.js

```diff
@@ -55,12 +55,14 @@ $(document).ready(function() {
   function submitStatusUpdate() {
     const data = new FormData();
     const message = $('#textarea');
+    const file = $("#file-upload").prop("files")[0];
```

Comment:
`file-upload` -> could we add a size check 
Don't want stuff crashing at our end (same size as grant image upload )

Reply:
I made it 3 MB. Not sure what the grant image upload size is.